pnpm-workspace.yaml

```diff
@@ -1,4 +1,4 @@
 packages:
   - 'packages/*'
-  - 'pnpm-sync-api-demo/*'
-  - 'pnpm-sync-cli-demo/*'
+  - 'pnpm-sync-api-demo/**'
+  - 'pnpm-sync-cli-demo/**'
```

Comment:
Suggest to eliminate `*` and `**` in this file and just provide an explicit list of the projects in the monorepo.  This will make it easier for people to understand what the projects are.